### ISSUE TITLE
nvda doesn't read SuggestionAvailableAlertText of PeoplePicker fluent UI field.

### ISSUE THREAD

**Issue by srinivasNithin** [nvaccess/nvda#11148]
### Steps to reproduce:
Turn on nvda and try to access this codepen page https://codepen.io/srinivas-nithin/pen/QWjxjNr?editors=0010 in chrome.
When you click on people picker dropdown , dropdown expands and suggestions are listed.

### Actual behavior:
nvda read the dropdown box details but doesn't announce "People Picker Suggestions available" text when suggestion are listed.

### Expected behavior:
nvda should be announcing that People Picker Suggestions available are available as suggested by the field property "suggestionsAvailableAlertText"

#### NVDA version:
2020.1.20167
#### Windows version:
windows 10

**Comment by DrSooom** [comment]
@srinivasNithin: Please specify the Windows 10 version and list all web browsers incl. version where this issue occurs. Thanks.

**Comment by srinivasNithin** [comment]
@DrSooom 
Here are the details
1. Windows OS : Microsoft Windows 10 Enterprise 10.0.18363
2. Browsers :  Google Chrome 81.0.4044.138 (Official Build) (64-bit)
                   :  Microsoft Edge Version 81.0.416.72 (Official build) (64-bit)

**Comment by srinivasNithin** [comment]
@DrSooom Is there any update on this?

**Comment by DrSooom** [comment]
@srinivasNithin: I just asked for the information, that was once missing, so that others could reproduce this issue easier.

**Comment by Adriani90** [comment]
@srinivasNithin are you still able to reproduce this issue in NVDA 2023.1?

**Comment by Adriani90** [comment]
We didn't get any updates upon request since many years, I am closing as abandoned. if you are still having this issue, please provide a working minimal test case and we can reopen.
